freqtrade/persistence/models.py

```diff
@@ -371,7 +371,7 @@ def close_date_utc(self):
         return self.close_date.replace(tzinfo=timezone.utc)
 
     @property
-    def enter_side(self) -> str:
+    def entry_side(self) -> str:
```

Comment:
Please add a (temporary) compatibility layer here (for strategies using this already with develop).

This means - a property named `enter_side` (which calls `entry_side`). 
It should have the docstring of "DEPRECATED, please use entry_side` - and a TODO with "remove me after 2022.5".

Reply:
I'm impatient - fixed it myself ... :laughing: